src/lib.rs

```diff
@@ -1049,6 +1049,18 @@ impl GooseAttack {
             }
         }
 
+        // Create a channel allowing GooseUser threads to log errors.
+        let (all_threads_logger, logger_receiver): (
+            mpsc::UnboundedSender<Option<GooseDebug>>,
+            mpsc::UnboundedReceiver<Option<GooseDebug>>,
+        ) = mpsc::unbounded_channel();
+
+        // Launch a new user.
+        let logger_thread = tokio::spawn(logger::logger_main(
+            self.configuration.clone(),
+            logger_receiver,
+        ));
```

Comment:
Is there some effort to setup this process?

Reply:
It consumes minimal system resources. But really it should be wrapped in Option<>.

Reply:
fixed